packages/plasma-core/src/components/Basebox/Basebox.tsx

```diff
@@ -14,7 +14,7 @@ export type ControlProps = {
     /**
      * Метка-подпись к элементу
      */
-    label?: string | number;
+    label?: React.ReactNode;
```

Comment:
```suggestion
    label?: string | number | React.ReactNode;
```
наверное лучше так, чтобы было по аналогии с description. Кстати, а чем не подходит description для таких целей?